R/mainTests.r

```diff
@@ -1498,6 +1953,39 @@ filter_group_duplicate <- function(meta) {
   return(output)
 }
 
+# whitespace_filters -------------------------------------
+# check no filter labels have leading or trailing whitespace.
+
+whitespace_filters <- function(data, meta) {
+  filters <- meta %>%
+    filter(col_type == "Filter") %>%
+    pull(col_name)
+
+  test <- data %>%
+    select(all_of(filters), any_of(as.character(geography_matrix[, 2]))) %>%
```

Comment:
I think you're missing some of the geography columns out here? Should it be `geography_matrix[, 2:6]`? Though changing that will introduce NAs, so they might need removing if you do?

Reply:
This might work? 
`select(all_of(filters), any_of(c(as.character(geography_matrix[, 2:6]) %>% .[!is.na(.)]))) %>%`

Reply:
Thank you - this worked! Amended now :)